docs/Maths.md

```diff
@@ -47,6 +47,76 @@ Result:
 
 ![](matplotlib.png)
 
+### Using Pandas ###
+Create a plot using [pandas.DataFrame.plot](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.html)
```

Comment:
Could you mention the dependencies required / command to install them please?
```
pip install matplotlib pandas
```

Reply:
No problem, just add this in current commit